src/test/java/org/jabref/logic/formatter/bibtexfields/AddBracesFormatterTest.java

```diff
@@ -72,4 +72,9 @@ public void formatDoesNotRemoveBracesInBrokenString() {
     public void formatExample() {
         assertEquals("{In CDMA}", formatter.format(formatter.getExampleInput()));
     }
+
+    @Test
+    public void formatBoundaryCase() {
```

Comment:
Which boundary case is that?`"Boundary" is a very generic term. Is it the two-letter written in upper case case?

Reply:
Yes, you're right. It's too generic..
This unit test addresses the length boundary of the provided string (i.e.  first conditional statement of the `format` method, line 24 in `AddBracesFormatter`).
I'll adjust it :)

Reply:
Why not checking the code when the boundary is not matched? (Without double check) I would expect: below the boundary, hitting the boundary exactly, above the boundary.

Reply:
Yeah sure, but this has been checked by the already existing unit tests. Hitting the boundary exactly was the only case that was missing.